s3_management/prune_nightly.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+(
+  set -x
+  python manage.py > packages_to_delete
+)
+
+DRY_RUN=${DRY_RUN:-enabled}
+DRY_RUN_FLAG="--dryrun"
+if [[ ${DRY_RUN} = "disabled" ]]; then
+  DRY_RUN_FLAG=""
+fi
+
+for pkg in $(cat packages_to_delete); do
+  (
+    set -x
+    aws s3 rm ${DRY_RUN_FLAG} "${pkg}"
+  )
+done
```

Comment:
Hmm, why not delete it right from the .py script?

Reply:
I wanted to have an option to do a dry run for testing purposes and it turns out that boto3 does not have a dry run option.

Reply:
see: https://stackoverflow.com/questions/57040302/how-to-run-boto3-run-job-flow-in-a-dry-run